tests/integration/v4/forms/forms.v4.feature

```diff
@@ -0,0 +1,271 @@
+@oauth2Skip
```

Comment:
I don't *love* that we do oauth2skip here (copied from forms.feature) so maybe a thing to reconsider so we actually check permissions correctly :grimacing: